.gitignore

```diff
@@ -351,3 +351,5 @@ MigrationBackup/
 .ionide/
 appsettings.Development.json
 .DS_Store
+/src/SFA.DAS.Admin.Aan.Web/appsettings.json
+/src/SFA.DAS.Admin.Aan.Web/appsettings.json
```

Comment:
you have this twice, but you shouldn't need it at all, as this file should be gone, our altered to appsettings.Development.json

Reply:
Done